src/kernels/potentials/simplerestraint.h

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2023 Team Dissolve and contributors
+
+#pragma once
+
+#include "classes/interactionpotential.h"
+#include "kernels/potentials/base.h"
+
+// SimpleRestraintPotential functional forms
+class SimpleRestraintPotentialFunctions
```

Comment:
And here too, sorry!
```suggestion
// RestraintPotential functional forms
class RestraintPotentialFunctions
```